pkg/splunk/enterprise/clustermaster.go

```diff
@@ -127,7 +139,17 @@ func ApplyClusterMaster(client splcommon.ControllerClient, cr *enterprisev1.Clus
 		if err != nil {
 			return result, err
 		}
-		result.Requeue = false
+
+		// Master apps bundle push requires multiple reconcile iterations in order to reflect the configMap on the CM pod.
```

Comment:
In the future we can think about updating status of cr if this operation fails. **cr.Status.Phase** can be updated with other values so that we can differentiate between deployment done vs additional operations needed